pkg/bpf/bpffs.go

```diff
@@ -29,8 +30,13 @@ var (
 
 	// Set to true on first get request to detect misorder
 	lockedDown = false
+	once       sync.Once
```

Comment:
The whole point of `lockedDown` is to catch whether the map root prefix is changed after the map root has been read. My question is whether you saw an actual race because just using `once` doesn't change the fact that this will still not have memory barriers around the access of `lockedDown` os if there was a real race, this would not fix it.